### DISCUSSION TITLE
How do you recommend bringing Sidebase into an already made project?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [sidebase/sidebase#102] [Q&A] [answered]
What is the best way to integrate Sidebase's stack and config into an already-made Nuxt 3 project?

**Comment by zoey-kaiser** [comment, marked as answer]
Hi @CodyBontecou 👋 

That is an excellent question. The first step I would say is to look into our modules (https://sidebase.io/nuxt-auth/getting-started), these are some of the building blocks we use to get sidebase started!

If you want to work with other parts of our starter you would need to copy/paste over the files that you need (such as Prisma, tRPC or others!)

You can find a complete version of a "generated" application on this repo (https://github.com/sidebase/sidebase).

**Reply by CodyBontecou** [reply]
Excellent, thank you. I ended up bringing in the modules manually. It was helpful having the Sidebase repo to look over.
